atc/atccmd/command.go

```diff
@@ -1113,6 +1114,7 @@ func (cmd *RunCommand) gcComponents(
 		atc.ComponentCollectorVolumes:           gc.NewVolumeCollector(dbVolumeRepository, cmd.GC.MissingGracePeriod),
 		atc.ComponentCollectorContainers:        gc.NewContainerCollector(dbContainerRepository, cmd.GC.MissingGracePeriod, cmd.GC.HijackGracePeriod),
 		atc.ComponentCollectorCheckSessions:     gc.NewResourceConfigCheckSessionCollector(resourceConfigCheckSessionLifecycle),
+		atc.ComponentCollectorPipelines:         gc.NewPipelineCollector(dbPipelineLifecycle),
```

Comment:
Right now, archiving pipelines is feature flagged - so IMO either we should get rid of the feature flag, or only conditionally create the GC component

Reply:
Might make sense to get rid of the flag altogether now that archiving pipelines is complete (I think?)

Reply:
Forgot there was even a flag for this! I think we can remove it since the RFC was merged.